tests/helpers/package-cache.js

```diff
@@ -309,11 +300,14 @@ PackageCache.prototype = {
    * @param {String} manifest The contents of the manifest file to write to disk.
    */
   _writeManifest: function(label, type, manifest) {
+    process.chdir(this.rootPath);
```

Comment:
`this.rootPath === originalWorkingDirectory` unless the user manually passes in a `rootPath` argument. I'm comfortable with things erroring out if the user specifies a non-existent folder. It'll be the correct error message and everything. I don't know what else might break? User deletes `cwd`? This will be the least of your problems.

I'd rather the API be something like passing in a `rootPath`, but I don't care too much. This is "fine."